datasets/numer_sense/README.md

```diff
@@ -0,0 +1,182 @@
+- slot-filling
+---
+
+# Dataset Card for [Dataset Name]
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://inklab.usc.edu/NumerSense/
+- **Repository:** https://github.com/INK-USC/NumerSense
+- **Paper:** https://arxiv.org/abs/2005.00683
+- **Leaderboard:** https://inklab.usc.edu/NumerSense/#exp
+- **Point of Contact:** Author emails listed in [paper](https://arxiv.org/abs/2005.00683)
+
+### Dataset Summary
+
+NumerSense is a new numerical commonsense reasoning probing task, with a diagnostic dataset consisting of 3,145
+masked-word-prediction probes. The general idea is to mask numbers between 0-10 in sentences mined from a commonsense
+corpus and evaluate whether a language model can correctly predict the masked value.
+
+### Supported Tasks and Leaderboards
+
+The dataset supports the task of slot-filling, specifically as an evaluation of numerical common sense. A leaderboard
+is included on the [dataset webpage](https://inklab.usc.edu/NumerSense/#exp) with included benchmarks for GPT-2,
+RoBERTa, BERT, and human performance. Leaderboards are included for both the core set and the adversarial set
+discussed below.
+
+### Languages
+
+This dataset is in English.
+
+## Dataset Structure
+
+### Data Instances
+
+Each instance consists of a sentence with a masked numerical value between 0-10 and (in the train set) a target.
+Example from the training set:
+
+```
+sentence: Black bears are about <mask> metres tall.
+target: two
+```
+
+### Data Fields
+
+Each value of the training set consists of (1) a sentence with a number masked out with the `<mask>` token, and (2) a
+ground truth target value (both strings). The test sets include the masked sentences but without the ground truth
+values.
+
+### Data Splits
+
+The dataset includes the following pre-defined data splits:
+
+- A train set with >10K labeled examples (i.e. containing a ground truth value)
+- A core test set (`test_core`) with 1,132 examples (no ground truth provided)
```

Comment:
could you make it explicit that we use the empty string `""` for no label for this split ?